File: ABB-setup/dv-processing-rel_1.7/docs/source/assets/code_samples/c++/reading_data/sample16.cpp
#include <dv-processing/io/camera_capture.hpp>

int main() {
	// Open a DVXplorer camera
	dv::io::CameraCapture capture("", dv::io::CameraCapture::CameraType::DVS);

	// Configure event sensitivity to default. Other sensitivities available: VeryLow, Low, High, VeryHigh
	capture.setDVSBiasSensitivity(dv::io::CameraCapture::BiasSensitivity::Default);

	// Configure event-frame readouts per second (here variable 5000 FPS, the default value)
	// See detailed API documentation for other available values
	capture.setDVXplorerEFPS(dv::io::CameraCapture::DVXeFPS::EFPS_VARIABLE_5000);

	// Enable global hold setting (already the default)
	capture.setDVSGlobalHold(true);
	// Disable global reset setting (already the default)
	capture.setDVXplorerGlobalReset(false);

	return 0;
}

File: source_code/ABB-setup/dv-processing-rel_1.7/docs/source/assets/code_samples/python/reading_data/sample16.py
import dv_processing as dv

# Open a DVXplorer camera
capture = dv.io.CameraCapture(dv.io.CameraCapture.CameraType.DVS)

# Configure default event sensitivity. Other sensitivities available: VeryLow, Low, High, VeryHigh
capture.setDVSBiasSensitivity(dv.io.CameraCapture.BiasSensitivity.Default)

# Configure event-frame readouts per second (here variable 5000 FPS, the default value)
# See detailed API documentation for other available values
capture.setDVXplorerEFPS(dv.io.CameraCapture.DVXeFPS.EFPS_VARIABLE_5000)

# Disable global hold setting. Default is True
capture.setDVSGlobalHold(False)
# Enable global reset setting. Default is False
capture.setDVXplorerGlobalReset(True)
